crates/e2e/src/tests/restart.rs

```diff
@@ -298,6 +332,25 @@ fn validator_catches_up_across_epochs() {
         shutdown_height: epoch_length + 1,
         restart_height: 2 * epoch_length + 1,
         final_height: 3 * epoch_length + 1,
+        assert_skips: false,
+    };
+
+    let _state = run_restart_test(setup);
+}
+
+#[test_traced("WARN")]
+fn validator_catches_up_with_gap_of_three_epochs() {
+    let _ = tempo_eyre::install();
+
+    let epoch_length = 30;
+    let setup = RestartSetup {
+        node_setup: Setup::new()
+            .epoch_length(epoch_length)
+            .connect_execution_layer_nodes(true),
```

Comment:
Since this is skipping blocks in the CL, we must be connected at the EL level so that the EL can fetch blocks from its peers.